tenants.libsonnet

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    hashring: 'thanos-receive',
```

Comment:
let's give this hashring a more descriptive name as every hashring will be made of `thanos-receive`. maybe `default` as it is the catch-all